401-code-challenges/FizzBuzzTree/fizz-buzz.test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const BinaryTree = require('./fizz-buzz-tree');
+const fizzBuzz = require('./fizz-buzz-tree');
```

Comment:
`const { fizzBuzz } = require('./fizz-buzz-tree');` ORRRRR combine the two `const {BinaryTree, fizzBuzz} = require('./fizz-buzz-tree')`